spec/helpers/bulk_action_helper_spec.rb

```diff
@@ -32,4 +32,40 @@
       end
     end
   end
+
+  describe '#show_report_link?' do
+    let(:bulk_action) { create(:bulk_action, action_type: 'ChecksumReportJob') }
+
+    context 'when status completed and file exists' do
+      before do
+        allow(bulk_action).to receive(:status).and_return('Completed')
+        allow(File).to receive(:exist?).and_return(true)
+      end
+
+      it 'returns true' do
+        expect(helper.show_report_link?(bulk_action, Settings.checksum_report_job.csv_filename)).to be_truthy
+      end
+    end
+
+    context 'when status completed but file does not exist' do
+      before do
+        allow(bulk_action).to receive(:status).and_return('Completed')
+        allow(File).to receive(:exist?).and_return(false)
+      end
+
+      it 'returns false' do
+        expect(helper.show_report_link?(bulk_action, Settings.checksum_report_job.csv_filename)).to be_falsey
```

Comment:
another way to do this is:
```
subject { helper.show_report_link?(bulk_action, Settings.checksum_report_job.csv_filename) }

it { is_expected.to be false }

```

Reply:
does this mean you want me to change it?  Is it worth it?

Reply:
i think the current syntax is easier to read (for me at least)

Reply:
i have a slight preference for the way it is at present (seems clearer, though maybe it's less idiomatic?).  but i don't care that much one way or the other.